.eslintrc.js

```diff
@@ -0,0 +1,68 @@
+const picassoCrossDependenciesRestrictionsMap = {
```

Comment:
```suggestion
const forbiddenImports = {
```
or `forbiddenImportsByPackage`